src/main/java/frc/robot/RobotContainer.java

```diff
@@ -50,14 +52,15 @@ public RobotContainer() {
     }
 
     private void configureDefaultCommands() {
-        swerve.setDefaultCommand(new HolonomicDrive(swerve, xbox::getLeftY, () -> -xbox.getLeftX(), xbox::getRightX));
+        shooter.setDefaultCommand(new Shoot(shooter, () -> 8));
+//        swerve.setDefaultCommand(new HolonomicDrive(swerve, xbox::getLeftY, () -> -xbox.getLeftX(), xbox::getRightX));
     }
 
     private void configureButtonBindings() {
-        a.whenPressed((Runnable) Robot::resetAngle);
-        b.toggleWhenPressed(new StopClimber(climber));
-
-        a.and(b).and(y).toggleWhenActive(new AdjustAngle(climber));
+//        a.whenPressed((Runnable) Robot::resetAngle);
+//        b.toggleWhenPressed(new StopClimber(climber));
+//
+//        a.and(b).and(y).toggleWhenActive(new AdjustAngle(climber));
```

Comment:
1. Don't commit commented-out code.
2. If this code needs to be deleted, delete it in another branch – this branch should only contain changes related to the shooter.